### ISSUE TITLE
AWS Staging: Appointments and requests canceled by patient show Canceled by Facility

### ISSUE THREAD

**Issue by beth-wodzinski** [department-of-veterans-affairs/va.gov-team#37732]
Steps to Reproduce:
1. Make an appointment in VAOS.
2. Return to the appointment listing. Find the appointment you just made. Display the detail.
3. Click the Cancel link. 
Expected result: appointment is canceled by patient. 
Actual result: vaos says "Facility canceled your appointment. If you want to reschedule, call us or schedule a new appointment online." 

4. Return to the main page. Display cancelled appointments. Find the appointment in question and display details.
Expected result: appointment was canceled by patient.
Actual result: vaos says "Facility canceled your appointment. If you want to reschedule, call us or schedule a new appointment online." 

End user impact: user is confused by incorrect information. 

Tested in SQA, with user jacqueline.morgan@id.me.![CancelMessage.png](https://images.zenhubusercontent.com/5f2d82cd1166696dbe88379c/38fa5285-e6d2-4d0c-afef-34186c2711a9)

Also this will have downstream impacts in VSSC reports  
![image.png](https://images.zenhubusercontent.com/5f2d82e4a65f7435058cfd75/e4b73d94-6f2e-4c4f-920b-b22e65c7dead)

**Comment by jenniemc** [comment]
I used Ola.Edgecombe@id.me optometry, cheyenne to make a direct schedule appt. 

VAOS Service API is not returning cancelationReason correctly. Created a backend ticket #38009

**Comment by ldelacosta** [comment]
@marcynadeau - This is ready to be tested. Please let us know if there is an issue on your end. Thanks.

**Comment by beth-wodzinski** [comment]
This is fixed for direct schedule, but requests still show as "facility canceled" 

Tested with Cecil Morgan, Cheyenne Mental Health, for direct scheduling (passed) and Cecil Morgan, FTC Amputation for requests (still says "facility canceled").

**Comment by ldelacosta** [comment]
Waiting for this issue to be resolved before the team can be retested. 
https://issues.mobilehealth.va.gov/browse/VAOSR-3222

**Comment by ldelacosta** [comment]
@beth-wodzinski - VAOSR-3222 has been fixed. Please re-test this ticket and let us know if there are any issues. Thanks!

**Comment by beth-wodzinski** [comment]
Tested and validated. These are the scenarios I tried:

1. Cancel request in VAOS. Displays "you cancelled." 
2. Cancel request in SM, reason "at veteran request." Vaos displays "cancelled by facility." Per Marcy, I will enter this as a minor defect. 
3. Cancel request in SM, reason "could not reach veteran." Vaos displays "cancelled by facility."
4. Cancel request in SM, reason "other". Vaos displays "cancelled by facility."
5. Cancel direct schedule in VAOS; displays "you cancelled." 

I also tried cancelling community care requests, but due to a known defect, cancellation is not behaving properly with CC requests. 

So this is good to go. @ldelacosta
